end-to-end-tests/get-blur-areas.sh

```diff
@@ -0,0 +1,31 @@
+# Makes a POST request with an image as parameter and compares the response with a file's content.
+# First parameter is the host to make the request to.
+# Second parameter is image's name.
+# Third parameter is the name of the file containing the expected output.
+check_image() {
+  echo "get-blur-areas: Running test for image: $2"
+
+  # Get the blobstore upload url.
+  uploadUrl=`curl "$1blobstore-upload-url?forwardurl=/get-blur-areas"`
+
+  # POST the image.
+  response=`curl -X POST -H "Content-Type: multipart/form-data" -F "image=@end-to-end-tests/files/$2" "$uploadUrl"`
+
+  # Compare the response we got with the expected one.
+  # If they are different, fail the test.
+  expected=`cat end-to-end-tests/expected/get-blur-areas/$3`
+  if [ "$response" != "$expected" ];
+  then
+    echo "get-blur-areas: Test failed at image: $2"
+    echo "Returned: $response"
+    echo "Should return: $expected"
+    exit 1 
```

Comment:
How about running all test cases and checking in the end if any of them failed. In the current state if the test fails on 'MultiplePeople' you won't know if it will also fail on 'OnePerson' until you fix the 'MultiplePeople' test and rerun the whole thing. Knowing which of the test cases failed would be helpful to identify possible problems upfront.

Reply:
Done! Thank you.